corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_htmx.scss

```diff
@@ -0,0 +1,72 @@
+@mixin htmx-spinner($width) {
+  position: absolute;
+  height: $width;
+  width: $width;
+  $offset: $width / 2;
+  left: calc(50% - $offset);
+  top: calc(50% - $offset);
+}
+
+form.htmx-request {
+  .htmx-loading {
+    position: relative;
+    &::before {
+      content: " ";
+      @extend .spinner-border;
+      @extend .spinner-border-sm;
+      @include htmx-spinner(22px);
```

Comment:
Given the various `px` sizing is this going to look right if it gets used on alternate sized inputs e.g. `btn-sm` or `btn-lg`? Not that we do it often, just wondering out loud.